src/main/java/mate/academy/service/transaction/BalanceHandlerImpl.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.service.transaction;
+
+import mate.academy.dao.FruitDao;
+import mate.academy.dao.FruitDaoImpl;
+import mate.academy.model.FruitTransaction;
+
+public class BalanceHandlerImpl implements TransactionHandler {
+    private final FruitDao fruitDao;
+
+    public BalanceHandlerImpl() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Override
+    public void handleTransaction(FruitTransaction transaction) {
```

Comment:
You need in some place perform check for negative values for some operations.
<img width="1001" alt="image" src="https://github.com/mate-academy/jv-fruit-shop/assets/72043323/09ca6a98-55b4-4a6d-a3b9-9332c9073693">

If I create the next csv data:
```
type,fruit,quantity
b,banana,-20
b,apple,100
```

The report will be:
```
fruit,quantity
banana,-20
apple,100
```